test/e2e/resize_nodes.go

```diff
@@ -101,7 +104,8 @@ func groupSize() (int, error) {
 }
 
 func waitForGroupSize(size int) error {
-	for start := time.Now(); time.Since(start) < 4*time.Minute; time.Sleep(5 * time.Second) {
+	timeout := 4 * time.Minute
```

Comment:
I think it makes a lot of sense where the constant is used more than once.  My gut feel is that when it is only used inside one function, it is more helpful to have the value visible right there.  Happy to change, but that is MHO!

Of course, resizeNodeNotReadyTimeout is only used once, so per that rule it should be in-function.  I'm going to wave my hands here about how it is thematically related to resizeNodeReadyTimeout!